diff -up sqlalchemy-migrate-0.5.4/setup.py.bak sqlalchemy-migrate-0.5.4/setup.py
--- sqlalchemy-migrate-0.5.4/setup.py.bak	2010-04-13 17:48:58.007206349 -0400
+++ sqlalchemy-migrate-0.5.4/setup.py	2010-04-13 17:49:38.058331216 -0400
@@ -41,8 +41,8 @@ Migrate extends SQLAlchemy to have datab
 
     entry_points = """
     [console_scripts]
-    migrate = migrate.versioning.shell:main
-    migrate-repository = migrate.versioning.migrate_repository:main
+    sqlalchemy-migrate = migrate.versioning.shell:main
+    sqlalchemy-migrate-repository = migrate.versioning.migrate_repository:main
     """,
     test_suite = "nose.collector",
 )
