Index: sqlalchemy-migrate-0.5.1.dev-r479/setup.py
===================================================================
--- sqlalchemy-migrate-0.5.1.dev-r479.orig/setup.py
+++ sqlalchemy-migrate-0.5.1.dev-r479/setup.py
@@ -31,8 +31,8 @@ Migrate extends SQLAlchemy to have datab
 
     entry_points = """
     [console_scripts]
-    migrate = migrate.versioning.shell:main
-    migrate-repository = migrate.versioning.migrate_repository:main
+    sqlalchemy-migrate = migrate.versioning.shell:main
+    sqlalchemy-migrate-repository = migrate.versioning.migrate_repository:main
     """,
     test_suite = "nose.collector",
 )
