diff -up sqlalchemy-migrate-0.5.3/setup.py.bak sqlalchemy-migrate-0.5.3/setup.py
--- sqlalchemy-migrate-0.5.3/setup.py.bak	2009-04-16 10:52:18.000000000 -0700
+++ sqlalchemy-migrate-0.5.3/setup.py	2009-04-16 10:52:43.000000000 -0700
@@ -40,8 +40,8 @@ Migrate extends SQLAlchemy to have datab
 
     entry_points = """
     [console_scripts]
-    migrate = migrate.versioning.shell:main
-    migrate-repository = migrate.versioning.migrate_repository:main
+    sqlalchemy-migrate = migrate.versioning.shell:main
+    sqlalchemy-migrate-repository = migrate.versioning.migrate_repository:main
     """,
     test_suite = "nose.collector",
     tests_require = test_requirements,
