Index: setup.py
===================================================================
--- setup.py	(revision 444)
+++ setup.py	(working copy)
@@ -35,6 +35,7 @@
     entry_points = """
     [console_scripts]
     migrate = migrate.versioning.shell:main
+    migrate-repository = migrate.versioning.migrate_repository:main
     """,
     test_suite = "py.test.cmdline.main",
 )
Index: migrate/versioning/migrate_repository.py
===================================================================
--- migrate/versioning/migrate_repository.py	(revision 444)
+++ migrate/versioning/migrate_repository.py	(working copy)
@@ -4,7 +4,7 @@
 
 
 def usage():
-    
+
     print '''Usage: %(prog)s repository-to-migrate
 
 Upgrade your repository to the new flat format.
@@ -28,8 +28,8 @@
 def deleteDirectory(dirpath):
     print '    Deleting directory: %s' % dirpath
     os.rmdir(dirpath)
-    
 
+
 def migrate_repository(repos):
     print 'Migrating repository at: %s to new format' % repos
     versions = '%s/versions' % repos
@@ -42,11 +42,11 @@
         files = os.listdir(origdir)
         files.sort()
         for file in files:
-            
+
             # Delete compiled Python files.
             if file.endswith('.pyc') or file.endswith('.pyo'):
                 deleteFile('%s/%s' % (origdir, file))
-                
+
             # Delete empty __init__.py files.
             origfile = '%s/__init__.py' % origdir
             if os.path.exists(origfile) and len(open(origfile).read()) == 0:
@@ -69,10 +69,12 @@
 
         # Try to remove directory. Will fail if it's not empty.
         deleteDirectory(origdir)
-    
 
-if __name__ == '__main__':
+
+def main():
     if len(sys.argv) != 2:
         usage()
     migrate_repository(sys.argv[1])
 
+if __name__ == '__main__':
+    main()
