Index: sqlalchemy-migrate-0.4.5/setup.py
===================================================================
--- sqlalchemy-migrate-0.4.5.orig/setup.py
+++ sqlalchemy-migrate-0.4.5/setup.py
@@ -34,8 +34,8 @@ Migrate extends SQLAlchemy to have datab
 
     entry_points = """
     [console_scripts]
-    migrate = migrate.versioning.shell:main
-    migrate-repository = migrate.versioning.migrate_repository:main
+    sqlalchemy-migrate = migrate.versioning.shell:main
+    sqlalchemy-migrate-repository = migrate.versioning.migrate_repository:main
     """,
     test_suite = "py.test.cmdline.main",
 )
