Index: setup.py
===================================================================
--- setup.py
+++ setup.py	2010-07-31 23:40:30.923637842 +0200
@@ -32,8 +32,8 @@
     license = "MIT",
     entry_points = """
     [console_scripts]
-    migrate = migrate.versioning.shell:main
-    migrate-repository = migrate.versioning.migrate_repository:main
+    sqlalchemy-migrate = migrate.versioning.shell:main
+    sqlalchemy-migrate-repository = migrate.versioning.migrate_repository:main
     """,
     test_suite = "nose.collector",
 )
